recipes/nionui-tool/bld.bat

```diff
@@ -0,0 +1,11 @@
+set
+set QT_BASE_X64="C:\Qt\5.14\msvc2017_64"
+set PYTHON3_BASE_X64="C:\Miniconda36-x64"
+dir
+msbuild launcher\NionUILauncher.vcxproj
+dir dist\
+powershell "Get-ChildItem -Include *.pdb -Recurse | foreach { $_.Delete() }"
+powershell "Get-ChildItem launcher\x64\Release\imageformats -Include *d.dll -Recurse | foreach { $_.Delete() }"
+%PYTHON% setup.py bdist_wheel
+dir dist\
+%PYTHON% -m pip install --no-deps  --ignore-installed dist\nionswift_tool-%PKG_VERSION%-cp36.cp37.cp38-none-win_amd64.whl -vv
```

Comment:
please add an exit condition after each line: https://github.com/conda-forge/make-feedstock/blob/master/recipe/bld.bat#L5

Reply:
I decided to go a different route and download the pre-built applications. Building the applications within the conda-forge environments is not going to be possible; and they're standalone independent applications anyway (no linking to Python). So they are built on Appveyor and Travis and then integrated into the package in this script.

However, I'd like to (1) see the artifacts before this recipe goes live; (2) update from 4.0.1b1 to 4.0.1. Is there any way to see the artifacts?

Reply:
Unfortunately not yet.  Check out CFEP-13, which could relate to previewing artifacts.